bundle-workflow/src/run_bwc_test.py

```diff
@@ -10,19 +10,23 @@
 import sys
 
 from manifests.bundle_manifest import BundleManifest
+from system import console
 from system.temporary_directory import TemporaryDirectory
+from system.working_directory import WorkingDirectory
 from test_workflow.bwc_test.bwc_test_suite import BwcTestSuite
 from test_workflow.test_args import TestArgs
 
 
 def main():
     args = TestArgs()
+    console.configure(level=args.logging_level)
     with TemporaryDirectory(keep=args.keep) as work_dir:
         logging.info("Switching to temporary work_dir: " + work_dir)
-        os.chdir(work_dir)
-        bundle_manifest = BundleManifest.from_s3(
-            args.s3_bucket, args.build_id, args.opensearch_version, args.architecture, work_dir)
-        BwcTestSuite(bundle_manifest, work_dir, args.component, args.keep).execute()
+        with WorkingDirectory(work_dir) as cur_dir:
+            os.chdir(work_dir)
```

Comment:
This won't be needed if you are using `WorkingDirectory `

Reply:
haha thanks for that. Bunch of conflicts and I missed this.